src/libs/ValidationUtils.ts

```diff
@@ -317,7 +317,8 @@ function isValidDisplayName(name: string): boolean {
  * Checks that the provided legal name doesn't contain special characters
  */
 function isValidLegalName(name: string): boolean {
-    return CONST.REGEX.ALPHABETIC_AND_LATIN_CHARS.test(name);
+    const hasAccentedChars = Boolean(name.match(CONST.REGEX.ACCENT_LATIN_CHARS));
+    return CONST.REGEX.ALPHABETIC_AND_LATIN_CHARS.test(name) && !hasAccentedChars;
```

Comment:
Is it possible to combine 2 regexes into one?
Please add automated test cases including `À-ÿ`.

Reply:
better to keep the regex separate, makes the regex more usable. Manual testing is possible in the current scenario, automated test as mentioned in [guidelines](https://github.com/Expensify/App/blob/main/tests/README.md#when-to-write-a-test) is when manual testing is not possible. Would be easier to test À-ÿ manually.

Reply:
We already have automated test for `isValidPersonName`.
Similarly, I still suggest to add test cases in order to avoid regression in the future.

Reply:
I understand the concern is regression, if the PR was modifying [Action](https://github.com/Expensify/App#actions) or modifying lower js library e.g the ExpensiMark parser then adding automated tests along with manual testing would definitely add value. Automated tests are for only when manual testing is difficult e.g. test to verify an outcome after an authentication token expires (which normally takes two hours). The À-ÿ range can be covered quite easily when providing screenshots and videos.

Reply:
Why regex cannot be simplified as `A-Za-z`?
Is there any latin character which belongs to `{Script=Latin}` but doesn't belong to `\u00C0-\u017F` other than A-Z, a-z?